### PR TITLE
web-components

### PR DESCRIPTION
**Pull request by anvidmen**

### CODE REVIEW

File: staff/juandavid-omen/web-components/my-app-react/logic/tweet.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ */
+    let newDate = new Date;
+    newDate = newDate.toDateString();
+    call('GET', 'https://skylabcoders.herokuapp.com/api/v2/users', undefined,
+        { 'Authorization': `Bearer ${token}` },
+        (error, status, body) => {
+            if (error) return callback(error)
+
+            if (status === 200) {
+                callback()
```
Comment (lichab) [inline staff/juandavid-omen/web-components/my-app-react/logic/tweet.js:18]: this callback should not be here

File: staff/juandavid-omen/web-components/my-app-react/components/Home.sass [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.home
+    @extend %panel
+    
+    background-color: #017c7c
+    color: white
+    width: 100%
+    height: 100%
+    font-family: arial,sans-serif
+
+    & button
```
Comment (alex-martinez-jativa) [inline staff/juandavid-omen/web-components/my-app-react/components/Home.sass:10]: Juan David, you should add a class to the bottom and implements BEM

File: staff/juandavid-omen/web-components/my-app-react/components/Landing.sass [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.landing
+    background: linear-gradient(to left,rgb(96, 255, 136), rgb(8, 255, 255))
+    text-align: center
+    height: 10rem
+    justify-content: center
+    padding-top: 7rem
+    & a
```
Comment (alex-martinez-jativa) [inline staff/juandavid-omen/web-components/my-app-react/components/Landing.sass:7]: the same, add class to anchor and apply BEM

File: staff/juandavid-omen/web-components/my-app-react/logic/retrieve-news.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        })
+
+        callback(undefined, data)
+    }
+
+    xhr.onerror = function (error) {
+        callback(new Error(`network error: ${error}`))
+    }
+
+    xhr.send()*/
```
Comment (alex-martinez-jativa) [inline staff/juandavid-omen/web-components/my-app-react/logic/retrieve-news.js:63]: this is old code, you should clean this file.

File: staff/juandavid-omen/web-components/my-app/components/Home.sass [hunk 5]
```diff
@@ -0,0 +1,18 @@
+.home
+    @extend %panel
+    
+    background-color: #017c7c
+    color: white
+    width: 100%
+    height: 100%
+    font-family: arial,sans-serif
+
+    & button
```
Comment (alex-martinez-jativa) [inline staff/juandavid-omen/web-components/my-app/components/Home.sass:10]: Juan, you should to add class to the elements and use BEM to add style

File: staff/juandavid-omen/web-components/my-app-react/logic/retrieve-user.spec.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+    beforeEach(function () {
+        users.length = 0;
+        name = names.random();
+        surname = surnames.random();
+        email = `${name.toLowerCase().split(' ').join('')}${surname.toLowerCase().split(' ').join('').concat(Math.random())}@mail.com`;
+        password = passwords.random();
+    })
+    it("should return the information of a specified user", (done) => {
+        //Crear al usuario de estrangis
```
Comment (alex-martinez-jativa) [inline staff/juandavid-omen/web-components/my-app-react/logic/retrieve-user.spec.js:12]: comments are used for third parties.
This kind of comment is not appropriate

Comment (anvidmen) [reply staff/juandavid-omen/web-components/my-app-react/logic/retrieve-user.spec.js]: Alex, thanks for your comments, I hope this is all resolved now.

**Pull request merged**